source/Jobbr.Client/JobbrClient.cs

```diff
@@ -1,32 +1,33 @@
 ﻿using System;
-using System.Linq;
 using System.Net;
 using System.Net.Http;
-using System.Runtime.Serialization;
 using System.Text;
+using System.Threading.Tasks;
+
 using Jobbr.Server.WebAPI.Model;
 using Newtonsoft.Json;
 
 namespace Jobbr.Client
 {
-    public class JobbrClient
+    public class JobbrClient : IJobbrClient
     {
         private readonly HttpClient httpClient;
 
         public JobbrClient(string backend)
         {
-            this.Backend = backend + (backend.EndsWith("/") ? string.Empty  : "/");
+            this.Backend = backend + (backend.EndsWith("/") ? string.Empty : "/");
             this.httpClient = new HttpClient { BaseAddress = new Uri(this.Backend) };
         }
 
         public string Backend { get; }
 
-        public JobDto GetJob(long id)
+        public JobDto GetJob(long id) => this.GetJobAsync(id).Result;
```

Comment:
This can lead to potential deadlocks in UI frameworks or ASP.NET

Reply:
ConfigureAwait(false) might be also worth a change

Reply:
Yeah I know, but THe current implementation already uses `this.httpClient.SendAsync(request).Result);` since HttpClient does not support a sync method.
But nevertheless you are right, I'll change my `.Result` to `.ConfigureAwait(false).GetAwaiter().GetResult()`